### PR TITLE
CURA-3527 Use different local dir per version

### PR DESCRIPTION
**Pull request by LipuFei**

### CODE REVIEW

File: UM/Resources.py [hunk 1]
```diff
@@ -264,33 +265,34 @@ def __find(cls, resource_type, *args):
 
     @classmethod
     def __initializeStoragePaths(cls):
+        storage_dir_name = cls.ApplicationIdentifier + "-" + cls.ApplicationVersion
```
Comment (nallath) [inline UM/Resources.py]: I thought we were gonna use a subfolder for this?

**Comment by LipuFei** [comment]
@nallath Changed to sub-directories :)

**Comment by fieldOfView** [comment]
How does this work with the upgrade system?

**Comment by LipuFei** [comment]
@fieldOfView That will be finished in another JIRA issue. We separated it into small tasks.

**Pull request merged**